src/components/Header/Header.js

```diff
@@ -120,30 +108,17 @@ const Header = ({ logout = () => {}, currentUser = undefined }) => {
     </OfflineHide>
   )
 
-  const profileImageButton = isProfileImageButtonShowing ? (
+  const userIconButton = (
     <UserButton aria-label="User account dropdown">
-      <CurrentUserImg src={currentUser?.picture} alt="User picture" onError={handleImageError} />
-    </UserButton>
-  ) : null
-  const initialsButton =
-    isInitialsButtonShowing && !isProfileImageButtonShowing ? (
-      <UserButton aria-label="User account dropdown">
-        <UserCircle>
-          {currentUserFirstInitial}
-          {currentUserLastInitial}
-        </UserCircle>
-      </UserButton>
-    ) : null
-
-  const fallbackButton = (
-    <UserButton aria-label="User account dropdown">
-      <BiggerIconUser />
+      <UserIcon
```

Comment:
Much better! it should have been this way in the first place for simplicity if DRY didnt justify the abstraction on its own